src/ui/DynamicHierarchicalFacet/DynamicHierarchicalFacet.ts

```diff
@@ -612,7 +638,13 @@ export class DynamicHierarchicalFacet extends Component implements IDynamicHiera
     };
   }
 
-  private buildFacetHeader() {
```

Comment:
This method was shortened in favor of adopting a structure more similar to [`DynamicFacet`](https://github.com/coveo/search-ui/blob/master/src/ui/DynamicFacet/DynamicFacet.ts#L724)'s